__tests__/app.test.js

```diff
@@ -143,6 +144,62 @@ describe('GET', () => {
 
     });
 
+    describe('GET: /api/articles/:article_id/comments', () => {
+
+        describe('GOOD PATH:', () => {
+
+            test(`200: /api/articles/:article_id/comments responds with an array of comment objects: 
+            each having comment_id, votes, created_at, author, body and article_id properties`, () => {
+                return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then(({body: { comments }}) => {
+                    expect(comments.length).toBe(11);
+                    expect(comments).toBeSortedBy('created_at', { descending: true });
```

Comment:
As this is a different behaviour, it would be suitable to extract into its own test